SConstruct

```diff
@@ -355,6 +355,7 @@ SConscript([
   'system/proclogd/SConscript',
   'system/ubloxd/SConscript',
   'system/loggerd/SConscript',
+  'system/ui/SConscript',
```

Comment:
ewww. any way we can do this without a build step? it was so beautiful without one...

Reply:
also will the python packaging stuff support this for setup, reset, etc?

Reply:
Hmm maybe if we compile nanosvg for each arch ahead of time, and load the lib in python